lib/code_corps_web/controllers/comment_controller.ex

```diff
@@ -1,30 +1,41 @@
 defmodule CodeCorpsWeb.CommentController do
   use CodeCorpsWeb, :controller
-  use JaResource
 
-  import CodeCorps.Helpers.Query, only: [id_filter: 2]
+  alias CodeCorps.{Comment, User, Helpers.Query}
 
-  alias CodeCorps.Comment
+  action_fallback CodeCorpsWeb.FallbackController
+  plug CodeCorpsWeb.Plug.DataToAttributes
```

Comment:
We can move this  one into the router `:api` pipeline once everything is fully switched.